presentation/src/main/java/jjh/movie/app/presentation/base/BaseViewBindingFragment.kt

```diff
@@ -0,0 +1,32 @@
+package jjh.movie.app.presentation.base
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.viewbinding.ViewBinding
+
+abstract class BaseViewBindingFragment<VB : ViewBinding>(
+    private val bindingInflater: (inflater: LayoutInflater) -> VB,
+) : Fragment() {
+
+    private var _binding: VB? = null
+
+    protected val binding: VB
+        get() = _binding ?: error("Binding is null")
```

Comment:
이 변수는 왜 언더바일까요.

Reply:
https://colour-my-memories-blue.tistory.com/6 참고 요망

Reply:
ㅈㅇㄷ?

Reply:
```suggestion
    private var _binding: VB? = null
    val binding: VB get() = requireNotNull(_binding)
```
별건 아닌데 requireNotNull로도 표현 가능, 똑같이 안에서 IllegalArgumentException 던지는거라

Reply:
지리는데용..?

<img width="793" alt="스크린샷 2024-02-23 오후 6 26 26" src="https://github.com/ze-zeh/MovieApp/assets/58066704/17f808f4-68e0-4ae4-9e45-beefbb876c7e">

<img width="668" alt="스크린샷 2024-02-23 오후 6 26 53" src="https://github.com/ze-zeh/MovieApp/assets/58066704/f78461a8-ed53-4d58-bf00-f686c26c27dc">

Reply:
엇 다른거였네 ㅋㅅㅋ

Reply:
여기서는 binding이 null이어서 제대로 초기화가 안된거니까 IllegalStatementException이 더 적절하려나..?